package.json

```diff
@@ -20,7 +20,7 @@
   "hash-int": "^1.0.0",
   "instabug-reactnative": "^8.0.19",
   "moment": "^2.13.0",
-  "native-base": "^2.6.0",
+  "native-base": "^2.4.0",
```

Comment:
why do we change this native-base version?

Reply:
2.6 turned the hamburger and reload icons to question marks on iOS

Reply:
5398abb sets native-base back to ^2.6.0